admin/tools/provision/cloudmanager.py

```diff
@@ -441,6 +523,21 @@ def build_cloudconfig(self):
 host: {hostname_tpl}
 fqdn: {hostname_tpl}
 
+write_files:
+- path: "/tmp/mount_volume.sh"
+  permissions: "0544"
+  owner: "root"
+  content: |
+    #!/bin/sh
+    set -e
+    while [ ! -b /dev/vdb1 ] ;
+    do
+      sleep 2
+      echo "---WAITING FOR CINDER VOLUME---"
+    done
```

Comment:
I keep it like that for know, because this code should be handle soon using openstack-heat configuration files.